diagnostic.md

```diff
@@ -100,11 +128,22 @@ object.
 ```
 
 ```sh
-# your answer here
+curl --request PATCH \
+  -header "Content-Type: application/json" \
+  http://www.example.com/donuts/42\
+  --data '{
+    "name": "Krüller"
+  }'
 ```
 
 Write an AJAX request to change the donut on the server using JSON.
 
 ```js
-let changeDonut = /* your answer here */;
+let changeDonut = function (data){
+  return a.jax({
+    url: 'http://www.example.com/donuts/',
+    method: 'PATCH',
+    data
+  });
+};
```

Comment:
You need to hard code this data in, this assumes too much about our click handler and has some typos (namely a.jax)

```js
let changeDonut = function (donutID) {
  return $.ajax({
    url: 'http://example.com/donuts/' + donutID,
    method: 'PATCH',
    data: {
      "donut" {
        "name": "Krüller",
      }
    }
  });
};
```